test/Microsoft.Extensions.Options.Test/OptionsTest.cs

```diff
@@ -302,5 +302,26 @@ public static TheoryData Configure_GetsEnumOptionsFromConfiguration_Data
             Assert.Collection(expectedValues, assertions.ToArray());
         }
 
+        [Fact]
+        public void Options_StaticCreateCreateMakesOptions()
+        {
+            var options = Options.Create(new FakeOptions
+            {
+                Message = "This is a message"
+            });
+
+            Assert.Equal("This is a message", options.Value.Message);
+        }
+
+        [Fact]
+        public void OptionsWrapper_MakesOptions()
+        {
+            var options = new OptionsWrapper<FakeOptions>(new FakeOptions
+            {
+                Message = "This is a message"
+            });
+
+            Assert.Equal("This is a message", options.Value.Message);
+        }
```

Comment:
Should we add one test that does the override just to make sure the behavior is defined now?

```
services.AddOptions();
services.AddSingleton<IOptions<Foo>>(Options.Create<Foo>(myfoo)>);
```